src/esn.calendar.libs/app/services/shells/calendar-shell.js

```diff
@@ -801,6 +802,48 @@ require('./valarm-shell.js');
       });
     }
 
+    /**
+     * Returns the differences between this and the object to compare
+     * @param  {CalendarShell} that an event
+     *
+     * @return {Array} an array containing the keys of differences between events.
+     */
+    function getChanges(that) {
```

Comment:
We can make this method's usage more generic. The aim of this method to check the difference properties beween 2 events. So, how about rename the method to: `difference` like [lodash](https://lodash.com/docs/4.17.15#difference) method name